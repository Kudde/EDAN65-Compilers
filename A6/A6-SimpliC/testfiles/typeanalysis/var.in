int fun(int a, int b) {
  return 42;
}

int main() {
  fun(3,4,5);
}
