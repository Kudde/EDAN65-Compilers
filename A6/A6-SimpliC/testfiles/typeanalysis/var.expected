Error at line 6: incorrect number of arguments
